src/Actions/Printing.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace FluxErp\Actions;
+
+use FluxErp\Http\Requests\PrintingRequest;
+use FluxErp\View\Printing\PrintableView;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\View\Factory;
+use Illuminate\View\View;
+
+class Printing extends FluxAction
+{
+    public \FluxErp\Printing\Printable $printable;
+
+    public Model $model;
+
+    public function boot($data): void
+    {
+        parent::boot($data);
+        $this->rules = (new PrintingRequest())->rules();
+
+        $this->model = $this->data['model_type']::query()->whereKey($this->data['model_id'])->firstOrFail();
```

Comment:
added a $this->validate() before. Every printing request is validated, if you want it or not. but this ensures that all needed data for printing is present. also changed the request because model type and model id are both required here…